aten/src/ATen/native/vulkan/VulkanOps.cpp

```diff
@@ -279,6 +274,44 @@ void conv2d_depthwise(
   vkDestroyDescriptorSetLayout(device, descriptorSetLayout, nullptr);
 }
 
+void conv2d_depthwise(
+    VulkanTensor& output,
+    const VulkanTensor& input,
+    const VulkanTensor& weight,
+    const c10::optional<float*> bias,
```

Comment:
You could consider c10::optional<const float*> since the memory location bias is pointing to is not supposed to change.  If you run into compilation issues because c10::optional does not provide implicit conversion constructors to convert from float* to const float* then please skip this suggestion.